src/Plugin/resource/DataProvider/DataProviderEntity.php

```diff
@@ -78,32 +78,19 @@ public function __construct(RequestInterface $request, ResourceFieldCollectionIn
       throw new InternalServerErrorException('The entity type was not provided.');
     }
     $this->entityType = $options['entityType'];
-    if (isset($options['bundles'])) {
+    if (!empty($options['bundles'])) {
       $this->bundles = $options['bundles'];
     }
+    else {
+      // If no bundles are passed, then assume all the bundles of the entity
+      // type.
+      $entity_info = entity_get_info($this->entityType);
+      $this->bundles = array_keys($entity_info['bundles']);
```

Comment:
I think this isn't always populated (e.g. for `user` entity where entity == bundle). 

`!empty($entity_info['bundles']) ? array_keys($entity_info['bundles']) : $entity_info['type']`